### PR TITLE
Popup(Dialog|MessageBox): friendly for accessibility

### PR DESCRIPTION
**Pull request by chikara-chan**
* [x] Make sure you follow Element's contributing guide ([中文](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.zh-CN.md) | [English](https://github.com/ElemeFE/element/blob/master/.github/CONTRIBUTING.en-US.md)).
* [x] Make sure you are merging your commits to `dev` branch.
* [x] Add some descriptions and refer relative issues for you PR.

**What's the problem**

[Dialog](http://element.eleme.io/#/en-US/component/dialog) and [MessageBox](http://element.eleme.io/#/en-US/component/message-box) is unfriendly for accessibility
1. Popup header `close` icon can't be focused.
2. When the popup is shown, with key `tab` pressed, the focus jump outside the popup, and move around the total page.

**What did I do**

1. Surround the `close` icon with button, rewrite the style.
2. Add document listener for `focusin` event to prevent focus jump outside the popup with `tab` key.

### CODE REVIEW

**Comment by baiyaaaaa** [comment]
You have not rebase your branch to dev correctly

**Comment by chikara-chan** [comment]
It has been resolved now!

File: src/utils/popup/popup-manager.js [hunk 1]
```diff
@@ -149,19 +149,41 @@ const PopupManager = {
...
+          ? topPopup.handleClose()
+          : (topPopup.handleAction ? topPopup.handleAction('cancel') : topPopup.close());
       }
     }
-  }
-});
+  });
+
+  // keep focusing inside the popup by `tab` key
+  document.addEventListener('focusin', function(event) {
```
Comment (QingWei-Li) [inline src/utils/popup/popup-manager.js]: 监听了事件但是没有注销？

Comment (chikara-chan) [inline src/utils/popup/popup-manager.js]: 原先的keydown事件也是全局监听没注销的

Comment (QingWei-Li) [inline src/utils/popup/popup-manager.js]: 想了下也对，这个只用注册一次

**Pull request merged**